zkstack_cli/crates/zkstack/src/commands/ecosystem/args/init.rs

```diff
@@ -205,4 +251,219 @@ pub struct EcosystemInitArgsFinal {
     pub skip_contract_compilation_override: bool,
     pub validium_args: ValidiumTypeArgs,
     pub support_l2_legacy_shared_bridge_test: bool,
+    pub bridgehub_address: H160,
+    pub deploy_ecosystem: bool,
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize, Parser)]
+pub struct RegisterCTMArgs {
+    #[clap(flatten)]
+    #[serde(flatten)]
+    pub ecosystem: EcosystemArgs,
+    #[clap(flatten)]
+    #[serde(flatten)]
+    pub forge_args: ForgeScriptArgs,
+    #[clap(long)]
+    pub update_submodules: Option<bool>,
```

Comment:
could you add `default_missing_value = "true"` ?

Reply:
`and are you sure, that non updating submodules should be a default behaviour?`

If we're calling init with update_submodules it would do it once, do we want to do it another 3 times when calling subcommands?

Reply:
No, I don't why?